### DISCUSSION TITLE
Anyway to install this package when composer.json is set to php:^7.3?

### DISCUSSION THREAD

**Discussion by vincent-lu** [spatie/ray#132] [Q&A]
Hi I'm trying to install this package on my local dev environment (which is running php 7.4), but the composer.json for this project is php:^7.3 since the server environment hasn't yet updated to 7.4.

I tried to change the composer.json to php:^7.3|^7.4 and then `composer require spatie/ray` but I'm still blocked due to ray's php version requirement.

Is there any way I can install this for this project?

**Comment by yuniorsk** [comment]
You should try `composer require spatie/ray --ignore-platform-reqs`

**Reply by freekmurze** [reply]
We don't recommend doing this, as our source code uses a lot of PHP 7.4 features
